provider_gsuite.go

```diff
@@ -0,0 +1,217 @@
+package jwtauth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/mitchellh/mapstructure"
+	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/jwt"
+	admin "google.golang.org/api/admin/directory/v1"
+	"google.golang.org/api/option"
+)
+
+// GSuiteProvider provides G Suite-specific configuration and behavior.
+type GSuiteProvider struct {
+	// Configuration for the provider
+	config GSuiteProviderConfig
+	// Google JWT configuration
+	jwtConfig *jwt.Config
+	// Google admin service
+	adminSvc *admin.Service
+	// Context for requests to Google admin APIs
+	ctx context.Context
+}
+
+// GSuiteProviderConfig represents the configuration for a GSuiteProvider.
+type GSuiteProviderConfig struct {
```

Comment:
Same comment about spacing.

Reply:
Done in ddda226fe68cf71f6c351ffe27cbef1ba09f109a.